### ISSUE TITLE
Duplicate keys detected: '[object Object]'. This may cause an update error.

### ISSUE THREAD

**Issue by tonraj** [voerro/vue-tagsinput#108]
When using existing-tags and v-model for pre selected tags then getting **Duplicate keys detected: '[object Object]'. This may cause an update error.** in Console.

**Comment by AlexMordred** [comment]
Please provide more details on how to reproduce this, a code example wouldn't hurt either.

**Comment by Erth0** [comment]
Same issue here as well, this is what I am using:

```html
<tags-input element-id="tags"
                                v-model="form.tags"
                                :existing-tags="existingTags"
                                :key="existingTags.key"
                                :typeahead="true"
                                placeholder="Isert Tags"></tags-input>
```
and this is the array of the existing tags:
```js
existingTags: [
                { key: 1, value: 'Web Development' },
                { key: 2, value: 'PHP' },
                { key: 3, value: 'JavaScript' },
            ],
```

**Comment by AlexMordred** [comment]
@Erth0 why do you have `:key="existingTags.key"`? Is the tags component inside a `v-for` loop and you're giving them all the same `:key` value? If so, this is not a problem with the component. Besides, `existingTags.key` would be `undefined`.

**Comment by tbergmueller** [comment]
I do get the same issues when using the Code from the [Usage Section](https://github.com/voerro/vue-tagsinput#usage). Using latest Version 2.4.0
However, the error disappeared, after I removed the attribute `element-id="tags"` from `<tags-input element-id="tags" ... >` . So the following may also resolve @Erth0 's issue:

```
<tags-input
                                v-model="form.tags"
                                :existing-tags="existingTags"
                                :key="existingTags.key"
                                :typeahead="true"
                                placeholder="Isert Tags"></tags-input>
```

Not sure what this does to the functionality of the component though?

**Comment by Huntedpix** [comment]
Hello !

I also do get the same issue when using the code from the Usage section. Removing the element-id also "fixed" the issue.

Just FYI :)

**Comment by AlexMordred** [comment]
Thanks for all the comments guys. I still can't reproduce the bug. I'm not getting this error on the component's demo page. I also followed the README and created the most basic page with the component - no errors either.

```html
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>

    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@voerro/vue-tagsinput@2.4.0/dist/style.css">
</head>
<body>
    <div id="app">
        <tags-input element-id="tags"
            v-model="selectedTags"
            :existing-tags="[
                { key: 'web-development', value: 'Web Development' },
                { key: 'php', value: 'PHP' },
                { key: 'javascript', value: 'JavaScript' },
            ]"
            :typeahead="true"></tags-input>
    </div>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/vue/2.6.12/vue.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/@voerro/vue-tagsinput@2.4.0/dist/voerro-vue-tagsinput.js"></script>

    <script>
        new Vue({
            el: '#app',
            components: { "tags-input": VoerroTagsInput },
            data: () => {
                return {
                    selectedTags: [
                        {
                            key: 'web-development',
                            value: 'Web Development'
                        },
                    ],
                };
            }
        });
    </script>
</body>
</html>
```

**Comment by vitordesousa** [comment]
@tonraj u can solve this issue? share ur result.. i'm with same problem.. 

```
			<tags-input element-id="tags"
			v-model.trim="keywords"
			:existing-tags="getKeywordsList"
			:key="getKeywordsList.key"
			@change="setMyKeywords($event)"
			:typeahead="true"
			:limit="7"
			:add-tags-on-comma="true"
			:add-tags-on-blur="false"
			:typeahead-max-results="5"
			:typeahead-hide-discard="true"
			:validate="invalidKeywords"
			placeholder="Escreva palavras-chave"
			wrapper-class="form-control2 form-control"
			></tags-input>
```

**Print above**
![image](https://user-images.githubusercontent.com/10253164/95948374-d26d5b80-0dc6-11eb-88d0-f96fc9aa9a13.png)

**Comment by tonraj** [comment]
> @tonraj u can solve this issue? share ur result.. i'm with same problem..
> 
> ```
> 			<tags-input element-id="tags"
> 			v-model.trim="keywords"
> 			:existing-tags="getKeywordsList"
> 			:key="getKeywordsList.key"
> 			@change="setMyKeywords($event)"
> 			:typeahead="true"
> 			:limit="7"
> 			:add-tags-on-comma="true"
> 			:add-tags-on-blur="false"
> 			:typeahead-max-results="5"
> 			:typeahead-hide-discard="true"
> 			:validate="invalidKeywords"
> 			placeholder="Escreva palavras-chave"
> 			wrapper-class="form-control2 form-control"
> 			></tags-input>
> ```
> 
> **Print above**
> ![image](https://user-images.githubusercontent.com/10253164/95948374-d26d5b80-0dc6-11eb-88d0-f96fc9aa9a13.png)

No I switched to another plugin.

**Comment by vitordesousa** [comment]
an friend solved for me... he said the same awser of @tbergmueller .. after i removed the `element-id` it's ok

**Comment by vitordesousa** [comment]
@tonraj if u no more use this plugin, please, close this issue.

**Comment by AlexMordred** [comment]
Fixed in v2.4.2
